src/sidebar/components/MarkdownEditor.tsx

```diff
@@ -181,49 +189,96 @@ function ToolbarButton({
   );
 }
 
+export type UserItem = {
+  username: string;
+  displayName: string | null;
+};
+
 type TextAreaProps = {
   classes?: string;
   containerRef?: Ref<HTMLTextAreaElement>;
-  atMentionsEnabled: boolean;
+  mentionsEnabled: boolean;
+  usersForSuggestions: UserItem[];
```

Comment:
`usersForMentions` might make it clearer how these users are used, though we'd have to note that you can still mention other users as well.